src/instructlab/lab.py

```diff
@@ -126,16 +126,26 @@ def cli(ctx, config):
     "Please do not use this option if you are planning to contribute back "
     "using the same taxonomy repository. ",
 )
+@click.option(
+    "--workspace",
```

Comment:
```suggestion
    "--workspace-path",
    type=click.Path(),
```

`workspace-path` would be more consistent with how the `taxonomy-path` option is named. It would also be good to set the type, since click has a path type.